build/Program.fs

```diff
@@ -400,8 +400,9 @@ let buildTargetTree () =
     "YarnInstall" ==>! "RunScript"
     "DotNetRestore" ==>! "RunScript"
 
-    "Clean" ==> "Format" ==> "RunScript"
-    ==>! "Default"
+    // Disable Fantomas because it generates invalid code F# code
+    // "Clean" ==> "Format" ==> "RunScript" ==>! "Default"
```

Comment:
You shouldn't disable the entirety of formatting because of one bug.
Please restore this and create a [.fantomasignore](https://github.com/fsprojects/fantomas/blob/master/docs-old/Documentation.md#ignore-files-fantomasignore) and include that one file where you have a problem.

Reply:
Well, if the file is not in the last commit then build process works.

Because, Ionide doesn't run the check on all the files but only the modified ones (from HEAD).

That's kind of a strange decision. I opened an issue to discuss it #1736